phlop/app/nvidia/ncu.py

```
@@ -0,0 +1,57 @@
+
+from phlop.dict import ValDict
+from phlop.proc import run, run_mp
+
+metrics = [
+    "all",
+    "l1tex__data_bank_conflicts_pipe_lsu_mem_shared_op_ld.sum",  # read
+    "l1tex__data_bank_conflicts_pipe_lsu_mem_shared_op_st.sum",  # wrte
+]
+
+
+def run(exe, events, output_file=None):
```

Ruff (0.8.0)

Redefinition of unused `run` from line 14

(F811)